annif/templates/home.html

```diff
@@ -122,60 +137,96 @@ <h2 class="mt-4">Results</h2>\
 });
 
 // Application, which uses the components above
-var app = new Vue({
+new Vue({
   delimiters: ["<%","%>"],
   el: '#app',
   data: {
-    results: results_,
-    errors: []
+    text: '',
+    project: '',
+    projects: [],
+    results: [],
+    problems: [],
+    loading: false
+  },
+  mounted: function() {
+    // TBD: we can add a button to reload the list of projects later
+    this.loadProjects();
   },
   methods: {
     clearResults: function() {
-      $('#results').empty();
       while (this.results.length > 0) {
         this.results.pop();
       }
     },
+    onProjectSelected(project) {
+      this.project = project;
+    },
+    onTextChanged(text) {
+      this.text = text;
+    },
+    loadProjects: function() {
+      var this_ = this;
+      $.ajax({
```

Comment:
This is the only use of jQuery now. Can be easily replaced now by Vue Resource, Axios, etc.